scripts/buildtools/webpack/plugins.js

```diff
@@ -30,87 +26,40 @@ function devMode() {
   });
 }
 
-function uglify() {
-  return new UglifyJsPlugin({
-    compress: {
-      warnings: false,
-      // Drop all console.* and Logger statements
-      drop_console: true,
-      drop_debugger: true,
-      pure_funcs: [
-        'Logger.trace',
-        'Logger.dir',
-        'Logger.time',
-        'Logger.timeEnd',
-        'Logger.group',
-        'Logger.groupEnd',
-        'Logger.assert',
-        'Logger.log',
-        'Logger.info',
-        'Logger.warn',
-        'Logger.deprecate'
-      ],
-    },
-    sourceMap: true,
-    comments: (node, comment) => {
-      // Remove other Okta copyrights
-      const isLicense = /^!/.test(comment.value) ||
-                      /.*(([Ll]icense)|([Cc]opyright)|(\([Cc]\))).*/.test(comment.value);
-      const isOkta = /.*Okta.*/.test(comment.value);
-
-      // Some licenses are in inline comments, rather than standard block comments.
-      // UglifyJS2 treats consecutive inline comments as separate comments, so we
-      // need exceptions to include all relevant licenses.
-      const exceptions = [
-        'Chosen, a Select Box Enhancer',
-        'by Patrick Filler for Harvest',
-        'Version 0.11.1',
-        'Full source at https://github.com/harvesthq/chosen',
-
-        'Underscore.js 1.8.3'
-      ];
-
-      const isException = exceptions.some(exception => {
-        return comment.value.indexOf(exception) !== -1;
-      });
-
-      return (isLicense || isException) && !isOkta;
-    }
-  });
-}
-
 function banner() {
   // Add a single Okta license after removing others
   const license = readFileSync(join(__dirname, '../../../src/widget/copyright.txt'), 'utf8');
   return new BannerPlugin(license);
 }
 
 function failOnBuildFail() {
-  return function() {
-    this.plugin('done', function(build) {
+  const FailOnBuildPlugin = function () {};
+  FailOnBuildPlugin.prototype.apply = function(compiler) {
+    compiler.hooks.done.tap('Fail on build', (stats) => {
+
       // webpack 3.x and karma-webpack combo will fail to treat seom missing assets as build failures
```

Comment:
remove the comment?

Reply:
Updated to current webpack version - it is still relevant with webpack@5 + karma-webpack